include/boost/geometry/algorithms/detail/buffer/get_piece_turns.hpp

```diff
@@ -246,20 +277,14 @@ class piece_turn_visitor
                 // and iterating in sync with them...
                 typedef detail::overlay::get_turn_info
                     <
-#if defined(BOOST_GEOMETRY_BUFFER_USE_SIDE_OF_INTERSECTION)
-                        buffer_assign_turn
-#else
                         detail::overlay::assign_null_policy
-#endif
                     > turn_policy;
 
-                turn_policy::apply(*prev1, *it1, *next1,
-                                    *prev2, *it2, *next2,
-                                    false, false, false, false,
-                                    the_model,
-                                    m_intersection_strategy,
-                                    m_robust_policy,
-                                    std::back_inserter(m_turns));
+                turn_policy::apply(the_model,
```

Comment:
Is there a reason why the order of arguments is changed? Previously the points were passed as the first arguments and after that strategies etc. Not that this is a problem, just pointing it out.